airbyte-integrations/connectors/source-relational-db/src/main/java/io/airbyte/integrations/source/relationaldb/AbstractDbSource.java

```diff
@@ -294,6 +294,18 @@ protected AutoCloseableIterator<AirbyteMessage> getFullRefreshStream(final Datab
     return getMessageIterator(queryStream, streamName, namespace, emittedAt.toEpochMilli());
   }
 
+  protected AutoCloseableIterator<AirbyteMessage> getFullRefreshStream(final Database database,
```

Comment:
Same deal here, repeats function definition

Reply:
The idea was to overload the function so that the original logic can be used for other connectors. The signature is different accordingly